test/io/test_io.py

```diff
@@ -140,16 +140,17 @@ def dumpconfig(configpath=None, env=None, stdin=None):
 
 
 @contextlib.contextmanager
-def run(configpath=None, stdin=None, env=None, port=None):
+def run(configpath=None, stdin=None, env=None, port=None, host=None):
     "Run PostgREST and yield an endpoint that is ready for connections."
     env = env or {}
     env["PGRST_DB_POOL"] = "1"
     env["PGRST_DB_POOL_TIMEOUT"] = "1"
+    host = host or "localhost"
 
     with tempfile.TemporaryDirectory() as tmpdir:
         if port:
             env["PGRST_SERVER_PORT"] = str(port)
-            env["PGRST_SERVER_HOST"] = "localhost"
+            env["PGRST_SERVER_HOST"] = host
```

Comment:
Right you change the default host for all tests that don't specify `host`.

To not accidentally change unrelated things, it'd be better to do:

```suggestion
            if host:
                env["PGRST_SERVER_HOST"] = host
```

And then you don't need the `host = host or ...` line above.